src/main/webapp/i18n/de/plagiarism.json

```diff
@@ -34,9 +34,9 @@
             "minimumScore": "Minimale Bewertung in %",
             "minimumScoreTooltip": "Berücksichtige nur Einreichungen mit einer Bewertung, die mindestens diesem Wert entspricht",
             "minimumSize": "Minimale Größe",
-            "minimumSizeTooltip": "Berücksichtige nur Einreichungen, deren Größe mindestens diesem Wert entspricht",
-            "minimumSizeTooltipModeling": "Berücksichtige nur Einreichungen, die mindestens so viele Modeling-Elemente wie der angegebene Wert haben",
-            "minimumSizeTooltipText": "Berücksichtige nur Einreichungen, die mindestens so viele Zeichen wie der angegebene Wert haben",
+            "minimumSizeTooltipTextExercise": "Consider only submissions that have at least as many different lines in comparison to the template as the specified value",
+            "minimumSizeTooltipProgrammingExercise": "Consider only submissions that have at least as many words as the specified value",
+            "minimumSizeTooltipModelingExercise": "Consider only submissions that have at least as many modeling elements as the specified value.",
```

Comment:
You did not translate these into German

Reply:
Just got translations from my translator @MarkusPaulsen :D